ui/app/src/App.tsx

```diff
@@ -37,8 +39,9 @@ function App(): ReactElement {
           backgroundColor: ({ palette }) => palette.background.default,
         }}
       >
-        {location.pathname !== SignInRoute && location.pathname !== SignUpRoute && <Header />}
-
+        <PluginRegistry pluginLoader={bundledPluginLoader}>
```

Comment:
```suggestion
        
          {location.pathname !== SignInRoute && location.pathname !== SignUpRoute && (
          <PluginRegistry pluginLoader={bundledPluginLoader}>
            <Header />
          </PluginRegistry>)}
```
Useless to load provider in certain pages